scripts/actions/mobskills/rotten_stench.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Rotten Stench
+-- Lowers physical and magic accuracy of enemies within a fan-shaped area originating from the caster.
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local power        = 50
```

Comment:
There are captures but I was unable to find the exact power of the effects. Would it require doing checkparam in game or is there a way to get it from captures?

Reply:
I would get a capture of this to check the duration. Accuracy down will show up with /checkparam but sadly magic acc down wont. It is probably fairly safe to assume they are the same.